### DISCUSSION TITLE
Rewritten FPU emulation for full 80-bit accuracy using software floating point

### DISCUSSION THREAD

**Discussion by Torinde** [PCBox/PCBox#60] [General]
One planned improvement was removed from the website:

https://github.com/PCBox/pcbox.github.io/blob/ac38e67ee595f7cca9e64490a41ff0b7984bd034/README.md#planned-improvements-for-the-future
> **Planned improvements for the future**
> Rewritten FPU emulation for full 80-bit accuracy using software floating point

Is it implemented already or the idea was dropped?

**Comment by pankoza2-pl** [comment]
The 80-bit SoftFloat FPU has been ported from 86Box (currently for interpreter only)

**Reply by Torinde** [reply]
Do you have an idea of how big is the SoftFloat performance impact?
- how much slower than native/host FPU speed?
- some example of what host CPU/FPU is needed to emulate which CPU/FPU (e.g. Ryzen 7700 3.8GHz is needed to match 486DX4 100MHz)?
